syft/format/syftjson/to_format_model.go

```diff
@@ -47,6 +49,16 @@ func ToFormatModel(s sbom.SBOM, cfg EncoderConfig) model.Document {
 	}
 }
 
+func getLayerOrderMap(s sbom.SBOM) map[string]int {
```

Comment:
I think we want to apply this same logic to SPDX and CycloneDX. It seems like having some of this logic only for syft json will mean we need to refactor it to apply to other formats.

Reply:
I was hoping to kick that can, but I can take another stab at that